spec/motion/serializer_spec.rb

```diff
@@ -28,32 +29,32 @@
   end
 
   describe "#weak_digest" do
-    subject(:weak_digest) { serializer.weak_digest(object) }
+    subject(:weak_digest) { serializer.weak_digest(component) }
 
     context "when the object can be serialized" do
-      let(:object) { [data] }
+      let(:marshal_dump) { { data: data } }
```

Comment:
I modified the specs to have something closer to the real use case, and because the code needs the object to respond to `marshal_dump`.
I could also have a more tolerant code, and leave the specs "as is", let me know what you think.